src/RootWithIntl.js

```diff
@@ -83,7 +84,6 @@ class RootWithIntl extends React.Component {
                         />
                         { (stripes.okapi !== 'object' || stripes.discovery.isFinished) && (
                           <ModuleContainer id="content">
-                            <OverlayContainer />
```

Comment:
@mkuklis have you tried dropdowns and modals? in https://github.com/folio-org/stripes-core/pull/1018/files I broke them because `OverlayContainer` was moved from `MainContainer`

Reply:
Yeah I tested couple of them and they seemed to work. Let me try to load couple more modules to double test it.

Reply:
<img width="1029" alt="Screen Shot 2021-04-09 at 09 05 41" src="https://user-images.githubusercontent.com/12440608/114131121-0d35f380-9913-11eb-9772-6e0b1473b00e.png">
<img width="1029" alt="Screen Shot 2021-04-09 at 09 06 18" src="https://user-images.githubusercontent.com/12440608/114131131-13c46b00-9913-11eb-9ee9-36e82634eda1.png">
seems dropdowns are ok, but not modals

Reply:
Thanks @NikitaSedyx! I will take another look at this.